pages/quests/index.tsx

```diff
@@ -54,6 +54,7 @@ const Quests: NextPage = () => {
                   <Input
                     placeholder="Keyword"
                     onChange={inputChange}
+                    textTransform="uppercase"
```

Comment:
uh good how about also deleting zenkaku space or hankaku space

Reply:
Are Japanese keywords assumed?

Reply:
Japanese is not assumed.